api/src/main/kotlin/com/openlattice/admin/AdminApi.kt

```diff
@@ -36,11 +38,14 @@ const val ID = "id"
 const val ID_PATH = "/{$ID}"
 const val NAME = "name"
 const val NAME_PATH = "/{$NAME}"
+const val WAREHOUSE_ID_PARAM = "WarehouseId"
+const val WAREHOUSE_ID_PATH = "/{${WAREHOUSE_ID_PARAM}}"
 
 const val JOBS = "/jobs"
 
-interface AdminApi {
+const val WAREHOUSES = "/warehouses"
```

Comment:
Our conventions are not consistent here, but what I've been doing is ... if it's part of the path, use `_PATH`, and if it's a param (path or otherwise), use `_PARAM`. So ...

```suggestion
const val WAREHOUSES_PATH = "/warehouses"
```